dev_tools/requirements/envs/format.env.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
I think the recommendation is to run the script using Python 3.10 (cc @mpharrigan). There's a docker version which you can try running if you are on a linux machine.